cmp/grid/Grid.js

```diff
@@ -222,7 +222,10 @@ class GridLocalModel extends HoistModel {
                 suppressSizeToFit: true // Without this the auto group col will get shrunk when we size to fit
             },
             autoSizePadding: 3, // tighten up cells for ag-Grid native autosizing.  Remove when Hoist autosizing no longer experimental,
-            editType: model.fullRowEditing ? 'fullRow' : undefined
+            editType: model.fullRowEditing ? 'fullRow' : undefined,
+            singleClickEdit: model.clicksToEdit === 1,
+            suppressClickEdit: model.clicksToEdit <= 0
```

Comment:
why don't we make suppress be != 1 && != 2

Reply:
Done - we had the doc comment saying as much, so seemed worth implementing that way